scripts/gen_openapi.sh

```diff
@@ -36,7 +36,7 @@ while read -r line; do
         codeLinesStarted=true
     elif [[ $line == *"databasePlugins:"* ]] ; then
         break
-    elif [ $codeLinesStarted = true ] && [[ $line == *"consts.Deprecated"* || $line == *"consts.PendingRemoval"* ]] ; then
+    elif [ $codeLinesStarted = true ] && [[ $line == *"consts.Deprecated"* || $line == *"consts.PendingRemoval"* || $line == *"consts.Removed"* ]] ; then
```

Comment:
Is this based off the comments in the registry like we previously had it? Or is this something new?

Reply:
Just out of curiosity b/c I don't remember anything that had said `PendingRemoval` or `Removed`

Reply:
They are no longer comments but actual struct values:

https://github.com/hashicorp/vault/blob/fd9cadb192e37c398f56c7fffe7f82832e166790/helper/builtinplugins/registry.go#L100-L103

What we are doing here (parsing Go code in shell) is kind of gross. I'm just fixing the script while we are still using it.

Reply:
oh okay! gotcha. At least the deprecation status is codified in here though.